lib/glific/clients/digital_green.ex

```diff
@@ -1,13 +1,193 @@
 defmodule Glific.Clients.DigitalGreen do
   @moduledoc """
-  Tweak and support some custom logic for a customer DigitalGreen
+  Tweak GCS Bucket name based on group that the contact is in (if any)
   """
 
+  import Ecto.Query, warn: false
+
+  alias Glific.{
+    Contacts,
+    Contacts.Contact,
+    Groups,
+    Groups.Group,
+    Repo
+  }
+
   @doc """
   Create a webhook with different signatures, so we can easily implement
   additional functionality as needed
   """
   @spec webhook(String.t(), map()) :: map()
-  def webhook("daily", fields), do: fields
+  def webhook("daily", fields) do
+    update_contact_field_value(
+      fields["contact_id"],
+      fields["contact_field_name"],
+      fields["increment_value"]
+    )
+
+    swap_groups(
+      fields["stage_one_group_name"],
+      fields["stage_two_group_name"],
+      fields["contact_field_name"],
+      fields["stage_one_upper_limit"],
+      fields["organization_id"],
+      fields["contact_id"]
+    )
+
+    swap_groups(
```

Comment:
Let's run a enum each with handle_stage function. Currently this code is not readable.